services/web/client/source/class/osparc/data/Resources.js

```diff
@@ -716,6 +735,37 @@ qx.Class.define("osparc.data.Resources", {
       };
     },
 
+    getCompatibleInputs: function(node1, portId1, node2) {
```

Comment:
I agree, but, at the moment, the caching mechanism is not flexible enough to deal with the "portsCompatibility" information, and since I need write access to it I rather put it in the same class.